common/flatpak-installation.c

```diff
@@ -1752,7 +1752,8 @@ flatpak_installation_install_full (FlatpakInstallation    *self,
                             ostree_progress, cancellable, error))
     goto out;
 
-  if (g_str_has_prefix (ref, "app"))
+  if ((flags & FLATPAK_INSTALL_FLAGS_NO_TRIGGERS) == 0 &&
```

Comment:
Nitpick: `!(flags & FLATPAK_INSTALL_FLAGS_NO_TRIGGERS)` would be a bit more readable.